Formula/nsq.rb

```diff
@@ -16,14 +16,15 @@ class Nsq < Formula
   end
 
   depends_on "go" => :build
-  depends_on "gpm" => :build
+  depends_on "dep" => :build
 
   def install
     ENV["GOPATH"] = buildpath
-    mkdir_p "src/github.com/nsqio"
-    ln_s buildpath, "src/github.com/nsqio/nsq"
-    system "gpm", "install"
-    system "make", "DESTDIR=#{prefix}", "PREFIX=", "install"
+    (buildpath/"src/github.com/nsqio/nsq").install buildpath.children
+    cd "src/github.com/nsqio/nsq" do
+      system "dep", "ensure"
+      system "make", "DESTDIR=#{prefix}", "PREFIX=", "install"
```

Comment:
You'll likely need to add a `prefix.install_metafiles` line here.

This ensures things like `README`, `LICENSE` and `CHANGELOG` where available are installed into the prefix. Normally Homebrew handles this automatically with a little magic, but when you relocate the entire contents of the `buildpath` to another directory for the install process as is common with `go`-using formulae that logic falls on its face & it needs a little manual nudging. It's a known issue.